public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -172,7 +172,7 @@ export function ToolbarActions({ dashboard }: Props) {
               onClick={() => {
                 const id = dashboard.onCreateNewPanel();
                 DashboardInteractions.toolbarAddButtonClicked({ item: 'add_visualization' });
-                locationService.partial({ editPanel: id });
+                locationService.partial({ editPanel: id, isNewPanel: true });
```

Comment:
if we updated onCreateNewPanel so also do `this.setState({ editPanel:  buildPanelEditScene(panel, true)} then I think we could skip this locationService call and the URL sync for isNewPanel (which feels wrong as that parm would not work anyway if doing a full page reload, as the panel would not be found)

Reply:
Ok, I see, let me try that!

Reply:
PR updated! Thanks Torkel for the feedback, now it looks much better